src/custom/state/orders/updater.ts

```diff
@@ -149,6 +165,15 @@ export function EventUpdater(): null {
         topics: eventTopics
       })
 
+      // only update lastCheckedBlock if no logs
+      if (logs.length === 0) {
+        // SET lastCheckedBlock = lastBlockNumber
+        dispatch(updateLastCheckedBlock({ chainId, lastCheckedBlock: lastBlockNumber }))
```

Comment:
u see :) you wouldn't need to replicate the dispatch and return in the middle, if u modeled the action as I suggested in the other PR.  There would be just a small map for the orders data. U wanted to use the fancy to use the `batchy-thingy`

Reply:
I'll create a few bulk actions tomorrow for orders, blocks

Reply:
still is not the end of the world, but this came to mind when reading. I think is nitypicky